plugins/tranquilo.py

```diff
@@ -1,23 +1,33 @@
 import re
 
+def __present_food(api, day_menu):
+    types = day_menu.find_all("h3")
+    descriptions = day_menu.find_all("p")[1:]
+    for (food_type, description) in zip(types, descriptions):
+        yield api.food(
+            food_type.get_text().strip(), 
+            description.get_text().strip())
+
 def food(api, date):
     if not api.is_current_week(date):
         return []
-    if not api.is_today(date):
-        raise NotImplementedError("Tranquilo currently only supports menus for today")
+    if not (api.is_today(date) or api.is_tomorrow(date)):
+        raise NotImplementedError("Tranquilo currently only supports menus for today and tomorrow")
```

Comment:
Should we really throw here? Most plugins don't support arbitrary dates so just saying "nope, no food" is probably better in this case?